### PR TITLE
SCAN-465/nesting and bundling

### PR DESCRIPTION
**Pull request by a-mikhailova**

### CODE REVIEW

File: docs/.vuepress/configs/navbar.ts [hunk 1]
```diff
@@ -10,6 +10,10 @@ export const navbar: Record<string, NavbarConfig> = {
       text: 'Networks',
       link: '/networks/'
     },
+    {
+      text: 'Products',
+      link: '/products/'
```
Comment (a-mikhailova) [inline docs/.vuepress/configs/navbar.ts:15]: it's a personal initiative to put food on the menu. Maybe I'm wrong.

File: docs/products/index.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 
-[**Minter**](./products/minter/index.md)
+[**Minter**](./minter/index.md)
 
-[**Scan**](./products/scan/index.md)
+[**Scan**](./scan/index.md)
 
-[**Wallet**](./products/wallet/index.md)
\ No newline at end of file
+[**Wallet**](./wallet/index.md)
```
Comment (a-mikhailova) [inline docs/products/index.md:11]: fixed links

File: docs/sdk-docs/methods.md [hunk 3]
```diff
@@ -2807,7 +2807,7 @@ const args: NestTokenArguments = {
   },
 };
 
-const result = await sdk.tokens.nestToken.submitWaitResult(args);
+const result = await sdk.tokens.nest.submitWaitResult(args);
```
Comment (a-mikhailova) [inline docs/sdk-docs/methods.md:2810]: method `nestToken` does not work

File: docs/sdk-docs/methods.md [hunk 4]
```diff
@@ -3312,7 +3312,7 @@ const args: UnnestTokenArguments = {
   },
 };
 
-const result = await sdk.tokens.unnestToken.submitWaitResult(args);
+const result = await sdk.tokens.unnest.submitWaitResult(args);
```
Comment (a-mikhailova) [inline docs/sdk-docs/methods.md:3315]: the same

File: docs/sdk-guides/examplesSDK.md [hunk 5]
```diff
@@ -82,9 +82,9 @@ export async function createToken(sdk, address, collectionId) {
 ////////////////////////////////////////////
 async function main() {
   const signer = await KeyringProvider.fromMnemonic(mnemonic);
-  const address = account.instance.address;
+  const address = signer.instance.address;
 
-  const sdk = createSdk(account);
+  const sdk = createSdk(signer);
```
Comment (a-mikhailova) [inline docs/sdk-guides/examplesSDK.md:87]: there's a typo - it doesn't work that way. here should be signer

**Comment by fend25** [comment]
Base branch should be develop.

We don't show products in the docs, at least this way and right now.

File: docs/.vuepress/configs/navbar.ts (hunk 1 above)
Comment (a-mikhailova) [reply docs/.vuepress/configs/navbar.ts]: I'm sorry - I hid it.

**Review by fend25 — APPROVED** [review]

**Pull request merged**